docs/design/editorial_design.md

```diff
@@ -0,0 +1,354 @@
+Designing an Edit API
+=====================
+
+# Design Overview
+The industry is full of timeline management software but nearly all of them include a fundamental set of tools for manipulation. The most well known being:
+
+- Overwrite
+- Trim
+- Cut
+- Slip
+- Slide
+- Ripple
+- Roll
+- Fill (3/4 Point Edit)
+
+OpenTimelineIO, while not seeking to become a fully realized NLE, could benefit from having a simple toolkit to help automate building/modifying timelines based on time rather than index alone.
```

Comment:
I think this could be rephrased to be closer to OpenTimelineIO's scope... maybe something like:

> OpenTimelineIO includes a suite of tools for operating on objects as raw data structures (append, etc), but is missing a library for performing higher level operations familiar to people from the editing world (Slip, 3/4 point edit, etc).  This proposal presents an API for working with OpenTimelineIO objects in those ways.

I don't think they need to be enumerated here, since you're going to be covering them in more detail later.